package.json

```diff
@@ -7,33 +7,29 @@
   "main": "dist/index.js",
   "browserslist": "last 2 versions",
   "dependencies": {
-    "@babel/plugin-transform-typescript": "^7.12.1",
-    "@babel/runtime": "^7.5.5",
-    "@editorjs/editorjs": "^2.19.0",
+    "@editorjs/editorjs": "^2.19.1",
     "@editorjs/header": "^2.6.0",
     "@editorjs/image": "^2.6.0",
     "@editorjs/list": "^1.6.0",
     "@editorjs/quote": "^2.4.0",
-    "@lhci/cli": "^0.6.1",
+    "@lhci/cli": "^0.7.0",
+    "@saleor/sdk": "github:mirumee/saleor-sdk#update_storefront_types",
```

Comment:
We should release a new version of sdk and use it here before merging PR to master

Reply:
Obviously